packages/server/src/api/notification/notification-alert.controller.ts

```diff
@@ -0,0 +1,314 @@
+          maxTime: moment().subtract(lookBackIndex - 1, 'days'),
+        })
+        .orderBy('observation.unit')
+        .getRawMany();
+
+      for (const row of rows) {
+        // if the unit is unknown or there are no symptoms, skip this entry
+        if (!unitSymptomCountsByDay[row.unit] || !row.symptoms) {
+          continue;
+        }
+
+        // parse out the symptoms
+        const symptoms = row.symptoms.split(',');
+        for (let symptom of symptoms) {
+          symptom = symptom.trim();
+          // if this symptom has not been encountered for the unit yet we need initialize the count structure
+          if (!unitSymptomCountsByDay[row.unit][symptom]) {
+            // this array is ordered/indexed from past to recent
+            unitSymptomCountsByDay[row.unit][symptom] = [0, 0, 0];
+          }
+          unitSymptomCountsByDay[row.unit][symptom][lookBackDays - lookBackIndex] += 1;
+        }
+      }
+    }
+
+    const numTrendingSymptomsByUnit: { [key: string]: number } = {};
+    Object.keys(unitSymptomCountsByDay).forEach((unit: string) => {
+      numTrendingSymptomsByUnit[unit] = 0;
+      // iterate over the daily count of each symptom for the unit
+      // if the symptom decreases or increases for every day then count
+      // it as trending
+      Object.keys(unitSymptomCountsByDay[unit]).forEach((symptom: string) => {
+        const symptomCountByDay = unitSymptomCountsByDay[unit][symptom];
+        let isIncreasingTrend = true;
+        let isDecreasingTrend = true;
+        for (let i = 0; i < symptomCountByDay.length; i++) {
+          if (i > 0) {
+            if (symptomCountByDay[i] >= symptomCountByDay[i - 1]) {
+              isDecreasingTrend = false;
+            }
+            if (symptomCountByDay[i] <= symptomCountByDay[i - 1]) {
+              isIncreasingTrend = false;
+            }
+          }
+        }
+        if (isIncreasingTrend || isDecreasingTrend) {
+          numTrendingSymptomsByUnit[unit] += 1;
+        }
+      });
+    });
+
+    for (const setting of userSettings) {
+      const alertData: AlertData = {};
+      Object.keys(numTrendingSymptomsByUnit).forEach((unit: string) => {
+        // only add data exceeding threshold
+        if (numTrendingSymptomsByUnit[unit] >= setting.threshold) {
+          alertData[unit] = numTrendingSymptomsByUnit[unit];
+        }
+      });
+      // only return records with actual alert data
+      if (!_.isEmpty(alertData)) {
+        output.push({
+          userSetting: setting,
+          alertData,
+        });
+      }
+    }
+    res.json(numTrendingSymptomsByUnit);
+  }
+
+  // return the number of symptoms per unit that are trending upwards or downwards over last 3 days
+  async getLowMusterRates(req: ApiRequest<OrgSettingParams>, res: Response) {
+    const output: OutputRecord[] = [];
+
+    const stats = await musterComplianceStatsByDateRange(
+      req.appOrg!,
+      req.appUserRole!,
```

Comment:
Since this endpoint is called by the notification service, I believe this will be full admin permissions, which should be fine, but it's something to keep in mind.  Only units that match the user's `UserRole` should also be part of the alert for each user as well.